packages/next-playwright/src/next-fixture.ts

```diff
@@ -0,0 +1,56 @@
+import type { Page, TestInfo } from '@playwright/test'
+import type { NextWorkerFixture, FetchHandler } from './next-worker-fixture'
+import { handleRoute } from './page-route'
+
+export interface NextFixture {
+  onFetch: (handler: FetchHandler) => void
+}
+
+class NextFixtureImpl implements NextFixture {
+  private fetchHandler: FetchHandler | null = null
+
+  constructor(
+    public testId: string,
+    private worker: NextWorkerFixture,
```

Comment:
This TypeScript syntax still trips me up, because of `private` it automatically does the `this.worker = worker` ceremony.

Reply:
Ditto. Though after writing it a few times it feels a bit better than repetitive `this.x = x`.